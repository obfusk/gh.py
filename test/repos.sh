#!/bin/bash
"$PY" gh.py repos obfusk \
  | jq '[.[]|select(.name|startswith("obfusk"))]'
