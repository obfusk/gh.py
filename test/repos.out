[
  {
    "name": "obfusk.coffee",
    "desc": "[TODO] obfusk.coffee - functional programming library for js/coffee",
    "link": "https://github.com/obfusk/obfusk.coffee"
  },
  {
    "name": "obfusk.github.io",
    "desc": "obfusk.github.io - /var/log/obfusk - hacking ⇒ ¬sleeping",
    "link": "https://github.com/obfusk/obfusk.github.io"
  },
  {
    "name": "obfusk.github.io-OLD",
    "desc": "[OLD]",
    "link": "https://github.com/obfusk/obfusk.github.io-OLD"
  },
  {
    "name": "obfusk.py",
    "desc": "obfusk.py - functional programming (& other tools) library for python (2+3)",
    "link": "https://github.com/obfusk/obfusk.py"
  },
  {
    "name": "obfusk.rb",
    "desc": "obfusk.rb - functional programming library for ruby",
    "link": "https://github.com/obfusk/obfusk.rb"
  }
]
